### ISSUE TITLE
Does this library actually work?

### ISSUE THREAD

**Issue by vincentpham13** [oauthjs/node-oauth2-server#753]
I can not see any code examples or documentation details, even got some weird errors.

`{
    "statusCode": 401,
    "message": "Unauthorized request: no authentication given"
}`

**Comment by killmenot** [comment]
@vincentpham13 Yes it works. You need to pass custom `authenticateHandler` to authorize handler or make sure there is authentication exists.

Please go though this stuff
https://github.com/oauthjs/node-oauth2-server/issues/358
https://github.com/oauthjs/node-oauth2-server/issues/264

There is an example of implementation
https://github.com/foundersandcoders/open-tourism-platform/tree/master

**Comment by jorenvandeweyer** [comment]
This library is not maintained anymore and has some security issues. You should use this [maintained successor](https://github.com/node-oauth/node-oauth2-server) instead!

**Comment by Lucyfurnice** [comment]
> This library is not maintained anymore and has some security issues. You should use this [maintained successor](https://github.com/node-oauth/node-oauth2-server) instead!

thank you.
